pulsar-client-cpp/python/pulsar/__init__.py

```diff
@@ -355,6 +355,7 @@ class Client:
     def __init__(self, service_url,
                  authentication=None,
                  operation_timeout_seconds=30,
+                 connection_timeout_ms=10000,
```

Comment:
please add the new parameter to the end of the parameters list, otherwise it will breaks the python runtime since it creates `Client` with positioned parameter but not by keyword parameter.